ReactNativeClient/lib/components/screens/encryption-config.js

```diff
@@ -131,6 +132,9 @@ class EncryptionConfigScreenComponent extends BaseScreenComponent {
 			try {
 				const password = this.state.passwordPromptAnswer;
 				if (!password) throw new Error(_('Password cannot be empty'));
+				const password2 = this.state.passwordPromptConfirmAnswer;
+				if (!password2) throw new Error(_('Confirm password cannot be empty'));
+				if (password !== password2) throw new Error(_('Passwords do not match'));
```

Comment:
As those are translated strings, please use the same text as on your CLI update: "Passwords did not match!"